crypto/ocsp/ocsp_vfy.c

```diff
@@ -204,8 +205,13 @@ static X509 *ocsp_find_signer_sk(STACK_OF(X509) *certs, OCSP_RESPID *id)
     /* Calculate hash of each key and compare */
     for (i = 0; i < sk_X509_num(certs); i++) {
         x = sk_X509_value(certs, i);
-        if (!X509_pubkey_digest(x, EVP_sha1(), tmphash, NULL))
+        if ((md = EVP_MD_fetch(x->libctx, SN_sha1, x->propq)) == NULL)
```

Comment:
hopefully x cant be NULL here..

Reply:
If it is, it crashes on the next step regardless.  Happy to add a NULL check though.

Reply:
It's coming from a stack of pointers.  If somebody pushes a NULL onto the stack, it's possible.  This seems like a very unlikely thing to be doing.